src/modules/site-v2/templates/strain/catalog.html

```diff
@@ -330,11 +330,14 @@ <h2 class="accordion-header" id="mappingSets">
                     lost_strains.push(pastedStrains[i])
                 }
             }
-            $("#copyPasteStrains").val('')
+            $("#copyPasteStrains, #search").val('')
+
+
             if (lost_strains.length > 0) {
                 set_alert(`The following strain(s) were not found ${lost_strains.join(", ")}`)
             }
-
+            
+            filtered_by_species.show()

```

Comment:
minor note -- the order of operations might be relevant here -- try swapping .show() and .sort() so that .show() renders the sorted content (this might avoid a quick UI flash of content being updated after shown).